sig/datadog/core/environment/cgroup.rbs

```diff
@@ -0,0 +1,19 @@
+module Datadog
```

Comment:
Would it be feasible to add these `rbs` files, the ones pertaining to non-security `ddtrace` code in a separate PR?
We can probably get those merged faster and slim this PR down a bit.

Reply:
Possibly, although the value is currently limited since they're here not to check the matching `.rb` file but to provide sigs for `datadog/security`. I'm not sure how this would interact with Sorbet (can it use RBS files right away?). Worth a test though.

Reply:
Sorbet does not yet use .rbs, but they can be converted to rbi using https://github.com/Shopify/rbs_parser . One challenge is that Sorbet does not validate that .rbi's match with the .rb code, e.g. it's possible that they drift apart and Sorbet will happily accept it.

I'm curious -- does steep complain about a .rb not matching its .rbs?

Reply:
(But +1 on splitting this and merging it separately from the rest of the PR)

Reply:
> I'm curious -- does steep complain about a .rb not matching its .rbs?

yes